Studenci/DybowskiMariusz_2019/src/Geometria/odcinek/OdcinekTest.java

```diff
@@ -0,0 +1,129 @@
+package Geometria.odcinek;
+
+import org.junit.*;
+
+public class OdcinekTest {
+//    @Test
+//    public void () {
+//        // Arrange
+//
+//        // Act
+//
+//        // Assert
+//
+//    }
+    @Test
+    public void odcinekConstructorDwaPunktyTest() {
+        // Arrange
+        Punkt punkt1 = new Punkt(1.0, 2.0);
+        Punkt punkt2 = new Punkt(2.0,3.0);
+
+
+        // Act
+        Odcinek odc1 = new Odcinek(punkt1, punkt2);
+        // Assert
+        Assert.assertEquals(1.0, odc1.getX1(), 0.001);
```

Comment:
Zwykle jeden assert na jeden test.

Reply:
Kierowałem się:
https://softwareengineering.stackexchange.com/questions/7823/is-it-ok-to-have-multiple-asserts-in-a-single-unit-test